tests/unit/AnnotatableTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace SilverLeague\IDEAnnotator\Tests;
+
+use SilverLeague\IDEAnnotator\Annotatable;
+use SilverLeague\IDEAnnotator\AnnotatePermissionChecker;
+use SilverLeague\IDEAnnotator\DataObjectAnnotator;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+
+class AnnotatableTest extends SapphireTest
+{
+
+    /**
+     * @var Annotatable
+     */
+    protected $extension;
+
+    public function setUp()
```

Comment:
setUp should be protected in PHPUnit 5.x

Reply:
I'm still not used to it being protected :|